invenio_communities/config.py

```diff
@@ -21,7 +21,9 @@
     'settings_privileges': '/communities/<pid_value>/settings/privileges',
     'members': '/communities/<pid_value>/members',
     'invitations': '/communities/<pid_value>/invitations',
+    'invitation_details': '/communities/<pid_value>/invitations/<invitation_id>'
```

Comment:
what is this needed for? it does not quite follow the pattern
in addition, there is already a generic `/requests/<request_id>` link